### ISSUE TITLE
Figure out how to shuffle data.

### ISSUE THREAD

**Issue by steventango** [oliverdiamond/continual-regression#15]
- https://github.com/haseebs/gvf-real-world/blob/main/dataloader.py#L63
- https://pytorch.org/docs/stable/_modules/torch/utils/data/sampler.html#RandomSampler
- https://github.com/jax-ml/jax/issues/12763
- https://github.com/Artur-Galstyan/jaxonloader
- https://github.com/jax-ml/jax/issues/14465
- https://stackoverflow.com/questions/64750139/performance-drop-when-slicing-jax-numpy-arrays
- https://stackoverflow.com/questions/68951669/is-there-a-way-to-speed-up-indexing-a-vector-with-jax
